asv_bench/benchmarks/rolling.py

```diff
@@ -113,4 +113,35 @@ def time_quantile(self, constructor, window, dtype, percentile,
         self.roll.quantile(percentile, interpolation=interpolation)
 
 
+class PeakMemFixed(object):
+
+    params = (['max', 'min'])
```

Comment:
It would be nice if all methods (and some other dtypes, constructors, etc.) could be used, but as is this test and the one below take almost a minute to run.  Some of the other methods (like median) can't even complete in the normal timeout of 60s.  Therefore, the scope of this test has been reduced to just the problem functions.

I can add other parameters if desired, but it will significantly increase the time the test takes.